src/app/models/students.model.ts

```diff
@@ -0,0 +1,14 @@
+export class Student {
```

Comment:
predefine default values in the constructor not to pass them every time

Reply:
Changed.